test/logic/models/content_test.dart

```diff
@@ -3,7 +3,7 @@ import 'package:collection/collection.dart';
 import '../../observer/observer.dart';
 import '../../test.dart';
 
-extension TestMap on Map<String, dynamic> {
+extension _TestMap on Map<String, dynamic> {
```

Comment:
you can remove the extension name, this will make it private

Reply:
Ah, even better, thanks. I didn't know that was possible.